libs/wire-subsystems/test/unit/Wire/MockInterpreters/InvitationCodeStore.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Wire.MockInterpreters.InvitationCodeStore where
+
+import Data.Id (InvitationId, TeamId)
+import Data.Json.Util (toUTCTimeMillis)
+import Data.Map (elems, (!?))
+import Data.Map qualified as M
+import Imports
+import Polysemy
+import Polysemy.State (State, get, gets, modify')
+import Unsafe.Coerce (unsafeCoerce)
+import Wire.API.User (InvitationCode (..))
+import Wire.InvitationCodeStore
+import Wire.InvitationCodeStore.Cassandra (mkInvitationCode)
+
+inMemoryInvitationCodeStoreInterpreter ::
+  forall r.
+  ( Member (State (Map (TeamId, InvitationId) StoredInvitation)) r,
+    Member (State (Map (InvitationCode) StoredInvitationInfo)) r
+  ) =>
+  InterpreterFor InvitationCodeStore r
+inMemoryInvitationCodeStoreInterpreter = interpret \case
+  InsertInvitation (MkInsertInvitation invitationId teamId role' createdAt' createdBy email name) _timeout -> do
+    code <- unsafeCoerce mkInvitationCode
```

Comment:
please remove this `unsafeCoerce`. I don't think it's necessary, especially without a type annotation or anything.